demo/deployment.yaml

```diff
@@ -16,7 +16,7 @@ spec:
     spec:
       containers:
         - name: server
-          image: ghcr.io/opensecuresupplychain/tkn-admission-controllers:0.0.2
+          image: us.icr.io/gitsecure/tkn-admcontroller:0.3.0
```

Comment:
Any reason we can't use the ghcr repo, are you hitting access rights still. ghcr is useful as we can all upload to there under GH team permissions